extern/sector-storage/sched.go

```diff
@@ -263,6 +265,9 @@ func (sh *scheduler) runSched() {
 		loop:
 			for {
 				select {
+				case <-sh.workerChange:
```

Comment:
Worker changes are unhandled?

Reply:
if this is 'okay' at least add a comment here explaining why

Reply:
It's a `chan struct{}`, this channel just pokes the scheduler to re-run, which is all that we need to do here